### DISCUSSION TITLE
I am getting this error while running the mysql in workstation server

### DISCUSSION THREAD

**Discussion by Priya-Devops2023** [learndevopsonline/learndevopsonline#1851] [Q&A]
Hi Raghu/Chaithanya
   [ centos@workstation ~/expense-ansible ]$ ansible-playbook -i mysql-dev.pdp2023.online, -e ansible_user=centos -e ansible_password=DevOps321 mysql.yml

PLAY [MySQL Setup] *************************************************************

TASK [Gathering Facts] *********************************************************
ok: [mysql-dev.pdp2023.online]

**TASK [mysql : Disable MySQL Module Default Version] ****************************
fatal: [mysql-dev.pdp2023.online]: FAILED! => {"changed": true, "cmd": "dnf module disable mysql -y", "delta": "0:00:16.201953", "end": "2024-03-09 13:48:09.456095", "msg": "non-zero return code", "rc": -6, "start": "2024-03-09 13:47:53.254142", "stderr": "Out of memory allocating 335544320 bytes!", "stderr_lines": ["Out of memory allocating 335544320 bytes!"], "stdout": "", "stdout_lines": []}**

PLAY RECAP *********************************************************************
mysql-dev.pdp2023.online   : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

54.161.50.246 | 172.31.46.173 | t3.small | https://github.com/Priya-Devops2023/expense-ansible.git

Kindly let me know how to solve this issue.

Thanks in advance 
Priya.

**Comment by chaithanya1812** [comment]
Hi @Priya-Devops2023 ,
What is your instance_type of mysql?  It's showing Out of memory
 Better to increase instance_type of mysql & try once..

## After posting the error don't stop your instances..

**Reply by Priya-Devops2023** [reply]
Hi Chaithanya
  Its t3.micro. I will change to t3.small n try.

**Reply by Priya-Devops2023** [reply]
Hi Chaithanya
   Thanks its working now :)
